### DISCUSSION TITLE
Pretty BibTex incompatibility

### DISCUSSION THREAD

**Discussion by huenchen** [RafaelGB/obsidian-db-folder#805] [Q&A]
whenever I am making changes to the table (i.e. selecting a new column to show) one can see those changes in the background, but whenever I close the window in which I did the changes, everythings goes back to the original state. Even when I hide certain columns an then try to slect some others to show, these hidden columns will reappear.
In the debug information it is saying "qinfo is not a function" and "q.error is not a function". I removed and reinstalled the plugin, but nothing changes.

**Comment by RafaelGB** [comment]
Hello @huenchen,
deactivate the developer option of logs

`q.error is not a function` should not be happening after the infusion of this constructor 2 versions ago

https://github.com/RafaelGB/obsidian-db-folder/blob/2ef9beee8254676395e8362fe040cee19d0efba0/src/services/Logger.ts#LL29C13-L29C24

**Comment by huenchen** [comment]
I activated the developer option of logs after I had the problems mentioned before. This even led to a very strange behavior, nothing was shown in the database at all. So I had to deactivate the Logger. But I re-installed the "db-folder" plugin, so I should not have old code. I changed the theme, to see, if this was the problem, but it was not. Now I can only thing of deactivating all the other plugins (124 of them), to see if they interfere.

**Comment by huenchen** [comment]
no change after deactivating all other plugins except dataview.

**Comment by RafaelGB** [comment]
try to activate the logs with trace level and then deactivate the logs.

This bug was introduced with 3.2.0 but fixed with 3.2.2

I can not reproduce it again with any device

Any way I improved the service for the next release

**Comment by RafaelGB** [comment]
thank you for all your test effort and sorry for the inconvenience

**Comment by huenchen** [comment]
don't be sorry, I really like your plugin and would be sorry, if I could not use it anymore. I did like you said and the logs sais "q.debug is not a function"
And then if I wanted to add a column, I could not choose any of the YAML Data, just add a really new one. After I switched the logging of, everything was like before, I could choose the YAML Data, see in the background that the new column was there, but after leaving that window no new column and all hidden columns were visible again.

**Comment by huenchen** [comment]
sometimes it is difficult to understand what i say, so I made screencasts and a screenshot:
[Screencasts](https://box.hu-berlin.de/d/d5b2d523a7ad48b8a4a5/)

**Comment by RafaelGB** [comment]
thanks for your screenshots!
I got it now.
Could you share the ddbb file? I will try to reproduce it

**Comment by huenchen** [comment]
yes, I put the newly created DB (daily_database) and an older one that was created with an older version, but this cannot. be changed either in the online folder with the sceenshots:
[Screencasts](https://box.hu-berlin.de/d/d5b2d523a7ad48b8a4a5/)

**Comment by RafaelGB** [comment]
https://user-images.githubusercontent.com/11924043/213569902-64f88dcd-d23a-4b09-8182-d91dd9e76e1f.mov


Seems fine under an empty folder

**Comment by huenchen** [comment]
yes in an empty folder it works for me too.

**Comment by huenchen** [comment]
but as soon, as I am putting a file in there the chosen YAML entry will not stay as a new column, but I can create a totally new column and that will stay.

**Comment by RafaelGB** [comment]
<img width="386" alt="Screenshot 2023-01-19 at 23 35 02" src="https://user-images.githubusercontent.com/11924043/213577487-71d162b0-c171-42c8-a33b-7c80eac4afa6.png">
 deactivate this flag and restart de app. Its the only thing I can imagine, some kind of big related with dataview

**Comment by huenchen** [comment]
no, sorry, i did it before and I just did it again, but id did not help. But thanks for all the time you put in!

**Comment by RafaelGB** [comment]
I will add this issue to the roadmap in case I can reproduce the problem so we do not forget about the problem

**Comment by huenchen** [comment]
I finally found the culprit. It was another plugin, that is interfering with db-folder. It's name is "Pretty BibTex", somehow it was not enough to deactivate the plugin. I had to manually remove it from the folder and restart Obsidian. Thank you for all your efforts. Sorry it took me so long, I had to remove one by one plugin of the 124, but it was worth it. Now I can use db-folder again.

**Comment by RafaelGB** [comment]
I am glad to read it! I will tag this issue as documentation to check incompatibilities with another plugins, great test job
